cohorts/views.py

```diff
@@ -2163,7 +2167,7 @@ def get_cohort_filter_panel(request, cohort_id=0, program_id=0):
 
 
 @login_required
-def cohort_files(request, cohort_id, limit=20, page=1, offset=0, build='HG38', access=None, type=None, files_only=False):
+def cohort_files(request, cohort_id, limit=25, page=1, offset=0, build='HG38', access=None, type=None, do_filter_count=True, files_only=False):
```

Comment:
We don't need both do_filter_count and files_only, since they do the same thing, so one of these should be dropped. Note his will require a refactor since other code was using files_only, but should be a quick fix (just search on files_only=True and replace with do_filter_counts=False)